docs/source/tutorials/filtsmooth/linear_bayesian_filtering_smoothing.ipynb

```diff
@@ -0,0 +1,4552 @@
+{
```

Comment:
<p>Perhaps add some explanation aka "This example thus requires a discrete, lti, Gaussian transition. In Probnum, this is the "DiscreteLTIGaussian" object, which needs to be initialised with the state transition, shift vector and process noise covariance matrix." </p><p> </p><p>What do you think? I suppose this would apply to the blocks below, too. </p><p> </p><p>It is also fine though if you dont wanna do it. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>